### PR TITLE
BETA: Register rifkimaulana.is-a.dev

### PR DESCRIPTION
**Pull request by ghost**
Added `rifkimaulana.is-a.dev` using the [dashboard](https://manage.is-a.dev).

### CODE REVIEW

File: domains/tkj1.json [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Mrfamou",
+        "email": "rifkiibul@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all",
+        "MX": "hosts.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/tkj1.json:9]:
```suggestion
        "MX": ["hosts.is-a.dev"]
```

File: domains/rifkimaulana.json [hunk 2]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Mrfamou",
+        "email": "rifkiibul@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all",
+        "MX": "hosts.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/rifkimaulana.json:9]:
```suggestion
        "MX": ["hosts.is-a.dev"]
```

File: domains/rifkiibul.json [hunk 3]
```diff
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "Mrfamou",
+        "email": "rifkiibul@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all",
+        "MX": "hosts.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/rifkiibul.json:9]:
```suggestion
        "MX": ["hosts.is-a.dev"]
```

File: domains/amoraclass.json [hunk 4]
```diff
@@ -0,0 +1,11 @@
+{
+    "owner": {
+        "username": "Mrfamou",
+        "email": "rifkiibul@gmail.com"
+    },
+    "record": {
+        "A": ["217.174.245.249"],
+        "TXT": "v=spf1 a mx ip4:217.174.245.249 ~all",
+        "MX": "hosts.is-a.dev"
```
Comment (WilliamDavidHarrison) [inline domains/amoraclass.json:9]:
```suggestion
        "MX": ["hosts.is-a.dev"]
```

**Review by wdhdev — CHANGES_REQUESTED** [review]

File: domains/rifkii.json [hunk 5]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Mrfamou",
+        "email": "rifkiibul@gmail.com"
+    },
+    "record": {
+        "CNAME": "12"
```
Comment (WilliamDavidHarrison) [inline domains/rifkii.json:7]: Please use a valid record.

**Review by wdhdev — CHANGES_REQUESTED** [review]

**Pull request closed without merging**